docs/examples.rst

```diff
@@ -48,3 +48,24 @@ SLURM Deployments
                           project="woodshole",
                           walltime="01:00:00",
                           queue="normal")
+
+
+
+SLURM Deployment: Low-priority node usage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+
+    from dask_jobqueue import SLURMCluster
+    cluster = SLURMCluster(processes=6, 
+                           threads=4, 
+                           memory="16GB", 
+                           project="co_laika", 
+                           queue='savio2_bigmem',
+                           env_extra=['export LANG="en_US.utf8"',
+                                      'export LANGUAGE="en_US.utf8"',
+                                      'export LC_ALL="en_US.utf8"', 
+                                      'qos="savio_lowprio"', 
+                                      'nodes="20"'])
```

Comment:
I'm curious, what does `nodes="20"` mean here?  Are you asking for 20 nodes?  If so then this might not be ideal.  The script should define getting only one node or one job.  We'll submit this script many times when you call `cluster.scale(100)` .

Reply:
sorry. Its meant to be ntasks-per-node. Removed and updated